c++/src/kj/filesystem-disk-unix.c++

```diff
@@ -72,6 +72,11 @@ namespace {
 #undef SEEK_HOLE
 #endif
 
+#if __ANDROID__
+// no DTTOIF function
```

Comment:
Could this be defined if we defined `_GNU_SOURCE` at the top?

Reply:
I'm not sure. _GNU_SOURCE has no meaning for Android AFAIK.